### PR TITLE
Add example configuration file

### PR DESCRIPTION
**Pull request by tobim**

### CODE REVIEW

**Comment by mavam** [comment]
Looks good to me...but what the heck is Jenkins complaining about?

**Comment by mavam** [comment]
I think Jenkins complains because the unit tests pick up the `vast.conf` and you've misspelled the `caf_flow` atom.

File: vast.conf [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+;; Format for printing individual log entries to the console (see file-format).
+;console-format="%m"
+
+;; Configures the minimum severity of messages that are written to the console
+;; (quiet|error|warning|info|debug|trace).
+;console-verbosity='info'
+
+;; Excludes listed components from logging (list of atoms).
+;component-blacklist=['caf', 'caf-flow']
```
Comment (mavam) [inline vast.conf]: `s/caf-flow/caf_flow/`

File: libvast/src/system/default_configuration.cpp [hunk 2]
```diff
@@ -71,7 +71,7 @@ caf::error default_configuration::merge_root_options(system::application& app) {
   content["vast"].as_dictionary().insert(options.begin(), options.end());
   auto default_fn = caf::defaults::logger::file_name;
   if (caf::get_or(*this, "logger.file-name", default_fn) == default_fn) {
-    path dir = get_or(*this, "vast.dir", defaults::command::directory);
+    path dir = get_or(*this, "vast.directory", defaults::command::directory);
```
Comment (Neverlord) [inline libvast/src/system/default_configuration.cpp]: I'm confused by this change. The command that sets this option uses `dir`: https://github.com/vast-io/vast/blob/master/libvast/src/system/default_application.cpp#L42. I don't see that changed in this commit, so I'd assume the command option no longer gets picked up correctly, no?

Comment (mavam) [reply libvast/src/system/default_configuration.cpp]: I remember having discussed this with @tobim at some point. We found that the long option shouldn't be a shortcut and that spelling it out is more intuitive.

Comment (Neverlord) [reply libvast/src/system/default_configuration.cpp]: Oh, I have no objections against the renaming. But I don't see a change to the command option.

Comment (mavam) [reply libvast/src/system/default_configuration.cpp]: Ah, agreed, that's still a missing piece.

File: libvast/src/system/spawn_node.cpp [hunk 3]
```diff
@@ -31,7 +31,7 @@ expected<scope_linked_actor> spawn_node(caf::scoped_actor& self,
                                         const caf::settings& opts) {
   // Fetch values from config.
   auto id = get_or(opts, "id", defaults::command::node_id);
```
Comment (mavam) [inline libvast/src/system/spawn_node.cpp]: BTW, should this also be `vast.id`?

Comment (tobim) [reply libvast/src/system/spawn_node.cpp]: Fixed in #453.

**Pull request merged**